homework_5/Methods/IsPalindrome.java

```diff
@@ -1,4 +1,22 @@
 package Methods;
 
+import java.util.Scanner;
+
 public class IsPalindrome {
+    public static boolean isPalindrome(String source){
+        String reverse = "";
+        for (int i = source.length() - 1; i >= 0; i--){
+            reverse += source.charAt(i);
+        }
+        if(source.toLowerCase().equals(reverse.toLowerCase()))
```

Comment:
if else statement can be simplified to -> `return source.toLowerCase().equals(reverse.toLowerCase());`